doc/13_spark_shell.md

```diff
@@ -0,0 +1,72 @@
+# Documentation
+
+## The Spark Shell and Spark Cassandra Connector
+
+These instructions were last confirmed with C* 2.0.11, Spark 1.2.1 and Connector 1.2.0-rc3
+
+### Setting up Cassandra
+
+The easiest way to get quickly started with Cassandra is to follow the instructions provided by 
+[Datastax](http://docs.datastax.com/en/cassandra/2.1/cassandra/install/install_cassandraTOC.html)
+
+### Setting up spark
+
+#### Download Spark
+
+Download a pre-built Spark from  https://spark.apache.org/downloads.html
+Untar the tar.gz downloaded with 
+
```

Comment:
I'd add somewhere a link to Spark documentation explaining how to run in standalone mode